### PR TITLE
Another Distillery recipe fix

### PR DESCRIPTION
**Pull request by Antifluxfield**
Only reduce the batch size if fluid amount is exceeding.
And this time I don't reduce the size of the normal recipes, because I found that we should reserve some space for overclocking.
@JohannesGaessler What about this one?

### CODE REVIEW

File: src/main/java/gregtech/common/GT_RecipeAdder.java [hunk 1]
```diff
@@ -562,6 +562,31 @@ public boolean addDistilleryRecipe(ItemStack aCircuit, FluidStack aInput, FluidS
...
+        			break;
+        		}
+        	}
+        	for (int i = tScale; i <= 10; i++) {
+        		if (aInput.amount % i == 0 && aDuration % i == 0 && aOutput.amount % i == 0) {
+        			tScale = i;
+        			break;
+        		}
+        	}
+        	aInput = new FluidStack(aInput.getFluid(), (aInput.amount + tScale - 1) / tScale);
```
Comment (JohannesGaessler) [inline src/main/java/gregtech/common/GT_RecipeAdder.java:583]: Why do you have this modifier for the input:
(aInput.amount + tScale - 1)

Comment (Antifluxfield) [reply src/main/java/gregtech/common/GT_RecipeAdder.java:583]: That's a ceil() after division. The input should not less than the exact amout of (amount / tScale).

**Review by draknyte1 — COMMENTED** [review]
You just need a function to find th smallest denominators of all factors. 
Ie: 16:4:4 becomes 4:1:1.

**Comment by Antifluxfield** [comment]
@draknyte1 
Not all recipes have such a good ratio: https://github.com/Blood-Asp/GT5-Unofficial/blob/unstable/src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java#L3005
Recipes like this can't be lossless. So I only test some small factors, trying to make it lossless; if that fail, just let it be.

**Pull request merged**